### PR TITLE
Updated Carl.gg Presence to fix ESLint issues, etc

### PR DESCRIPTION
**Pull request by dzlandis**
**Name of the service:**
Carl.gg

**URL's:**
https://carl.gg
https://docs.carl.gg

**Screenshots proving it still works after changes:**
Img Img Img Img Img

### CODE REVIEW

**Comment by dzlandis** [comment]
Adding the version change in the json file, was just making sure Codacy worked fine. 👍

**Comment by dzlandis** [comment]
Let me know if you'd like me to run prettier on it as well. I just haven't yet because I didn't wanna mess around with the original code too much as to keep ownership the same.

**Comment by dzlandis** [comment]
I didn't know renaming the branch would close the pull request. My bad!

File: websites/C/Carl-bot/presence.ts [hunk 1]
```diff
@@ -1,16 +1,14 @@
...
+}),
+browsingStamp = Math.floor(Date.now() / 1000);
 
 presence.on("UpdateData", () => {
   const presenceData: PresenceData = {
     largeImageKey: "carllogo",
     startTimestamp: browsingStamp
   };
 
-  if (document.location.hostname == "carl.gg") {
```
Comment (ririxidev) [inline websites/C/Carl-bot/presence.ts:13]: by this you are removing check and this second check to check hostname will not work lol, restore it and fix eslint issues

Comment (dzlandis) [reply websites/C/Carl-bot/presence.ts:13]: In my testing it appeared to work just fine but I'll edit it regardless.

File: websites/C/Carl-bot/presence.ts [hunk 2]
```diff
@@ -1,15 +1,14 @@
...
+browsingStamp = Math.floor(Date.now() / 1000);
 
 presence.on("UpdateData", () => {
   const presenceData: PresenceData = {
     largeImageKey: "carllogo",
     startTimestamp: browsingStamp
   };
-
+  
   if (document.location.hostname == "carl.gg") {
```
Comment (ririxidev) [inline websites/C/Carl-bot/presence.ts:12]:
```suggestion
  if (document.location.hostname === "carl.gg") {
```

**Review by rxri — CHANGES_REQUESTED** [review]

File: websites/C/Carl-bot/presence.ts (hunk 2 above)
Comment (dzlandis) [reply websites/C/Carl-bot/presence.ts]: Thanks, I thought I had fixed that.

**Review by Bas950 — APPROVED** [review]
LGTM

**Review by rxri — APPROVED** [review]

**Pull request merged**